five-minute-serverless-backend/README.md

```diff
@@ -0,0 +1,209 @@
+# Five Minute Serverless Backend
+*How to build a **serverless backend** for your web applciation and have it up in running on the cloud in minutes.*
+
+This tutorial takes you through 5 steps to building a backend for your web application:
+1. Setup your Binaris environment
+1. Create a serverless function for your backend code
+1. Update your function to work from a browser
+1. Call your function from frontend code
+
+## Setup your Binaris environment
+
+If you do not have a Binaris account, [sign up here](https://binaris.com/signup?t=XXXXXX). Binaris is free to use up to 400,000 GB-seconds a month. At the end of the sign-up process you will receive an account id and API key. Save them for the next steps.
```

Comment:
why put this in an example repo?
it will be a tech debt the minute we change stuff. better link to the right place with all the data imho

Reply:
Fixed